### PR TITLE
refactor(core): clear build config

### PR DESCRIPTION
**Pull request by EYHN**
remove build config

allowLocalWorkspace -> FeatureFlag
enablePreloading -> removed
enableNewSettingUnstableApi -> removed
enableExperimentalFeature -> removed
enableThemeEditor -> FeatureFlag

remove some unused code

### CODE REVIEW

**Comment by EYHN** [comment]
* **#8268** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8268?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/components/affine/setting-modal/general-setting/appearance/index.tsx [hunk 1]
```diff
@@ -98,51 +104,10 @@ export const AppearanceSettings = () => {
...
-              }))}
-              value={appSettings.windowFrameStyle}
-              className={settingWrapper}
-              width={250}
-              onChange={(value: AppSetting['windowFrameStyle']) => {
-                updateSettings('windowFrameStyle', value);
-              }}
-            />
-          </SettingRow>
-        ) : null}
```
Comment (pengx17) [inline packages/frontend/core/src/components/affine/setting-modal/general-setting/appearance/index.tsx:120]: why removed this and line 125?

Comment (EYHN) [reply packages/frontend/core/src/components/affine/setting-modal/general-setting/appearance/index.tsx:120]: I think these two options currently have no effect?

Comment (pengx17) [reply packages/frontend/core/src/components/affine/setting-modal/general-setting/appearance/index.tsx:120]: WOW what are these

**Review by pengx17 — APPROVED** [review]

**Pull request merged**